sheets/ASSETS.md

```diff
@@ -0,0 +1,73 @@
+# COMMON ASSET MANAGEMENT TASKS
+
+Although not strictly needed, the __asset__ files are used as a cross-check for when equipment
+is installed. This is to try and catch errors related to serial numbers, and also to allow
+using the asset numbers to further identify installed equipment. 
+
+The files in the __assets__ directory all have the same format. Each file represents an equipment
+type which also constrains where the equipment can be installed.
+For instance, only equipment in the __assets/sensors.csv__ file can be installed in the __install/sensors.csv__ file.
+
+Generally __assets__ should be entered in before they are added to files in the __installs__ directory.
+
+The presence of an asset "number" is an indication that the equipment is managed directly by __GeoNet__.
+
+[_The files were not built for asset tracking or inventory management, however this may change with further requirements_]
+
+## Files
+
+---
+* :page_with_curl: __assets/antennas.csv__
+* :page_with_curl: __assets/metsensors.csv__
+* :page_with_curl: __assets/radomes.csv__
+* :page_with_curl: __assets/receivers.csv__
+---
+* :page_with_curl: __assets/sensors.csv__
+* :page_with_curl: __assets/recorders.csv__
+* :page_with_curl: __assets/dataloggers.csv__
+---
+* :page_with_curl: __assets/cameras.csv__
+---
+
+## :one: _Updating an existing asset entry_
+
+Find the appropriate __asset__ file for the type of equipment.
+
+> ### :information_source: General requirements
+> 
+> * If changed, the make, model, or serial number of the assets to update.
+> * If changed, the equipment asset "numbers".
+> * If changed, any equipment notes for the updated assets.
+
+> ### :small_orange_diamond: Delta checks
+> 
+> * There can only be one asset with the same make, model, and serial number.
+> * There can only be one asset with the same asset "number".
+> * Any references to these assets in the __install__ directory files will also need to be consistent.
+
+## :two: _Adding a new asset entry_
+
+Find the appropriate __asset__ file for the type of equipment.
+
+> ### :information_source: General requirements
+> 
+> * The make, model, and serial number of the new `assets.
+> * Any optional asset "numbers" that may be available.
+> * Any extra equipment notes that may be appropriate.
+>
```

Comment:
Hi @ozym , can we please add something here specific to new GNSS equipment?
Something like:
```
for GNSS equipment, the name used in the "_model_" field need to be present in the IGS receiver and antenna table file: ftp://igs.org/pub/station/general/rcvr_ant.tab 
```

Reply:
That looks great, either I merge this into this PR or wait and merge it into master after this one has been sorted.

Reply:
It may be that these sheets don't live in this repo.

Reply:
thanks!!! whatever works best for you, I'll keep that branch and wait to see when this PR is sorted. 
You did a great job with these helpsheets, thanks heaps!

Reply:
relatively to the assets, that's ok. 
We will only need to make sure that the "model" names we use are IGS-compliant. 
I can add this note in the "GNSS.md" overall steps as well.